icu4c/source/i18n/unicode/numberformatter.h

```diff
@@ -1507,6 +1507,10 @@ class U_I18N_API NumberFormatterSettings {
      * All units will be properly localized with locale data, and all units are compatible with notation styles,
      * rounding precisions, and other number formatter settings.
      *
+     * \note If the usage() is set (draft API), the output unit **will be
```

Comment:
Draft APi problem: it's hard to suppress this paragraph based on U_HIDE_DRAFT_API - if I want to do this, I might need two full copies of the documentation? (One with, and one without, this paragraph?)

Reply:
Done. (I've taken that as a suggestion to remove the reference to "draft API".)